hassio-to-pubsub-source/cloud-functions/hass-to-bigquery/hass_to_bigquery.py

```diff
@@ -0,0 +1,41 @@
+import json
+import base64
+
+from google.cloud import bigquery
+
+PROJECT = "a2i2-ua-smart-home"
+DATASET = "event_data"
+TABLE = "events"
+
+bigquery_client = bigquery.Client(project=PROJECT)
+dataset = bigquery_client.dataset(DATASET)
+table_ref = dataset.table(TABLE)
+table = bigquery_client.get_table(table_ref)
+
+
+def hass_to_bigquery(data, context):
```

Comment:
We should omit the `cloud-functions` directory from this PR since it's specific to our project and not relevant to the add-on.